symplyphysics/laws/astronomy/angle_of_rotation_during_e_of_rotation_during_gravitational_maneuver.py

```diff
@@ -0,0 +1,45 @@
+from sympy import Eq, solve, atan
+from sympy.physics.units import gravitational_constant
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, angle_type)
+
+## Description
+## A gravitational maneuver is a purposeful change in the trajectory and flight speed of a spacecraft under the influence
+## of the gravitational fields of celestial bodies.
+## The angle of the gravitational maneuver depends on the aiming range, the mass of the planet and the velocity of the rocket relative to the planet.
+## The aiming range is the distance between the asymptote of the hyperbolic trajectory of the circumnavigation of the planet and its focus
+## coinciding with the center of the planet.
+
+## Law is: phi = 2 * arctg(G * M / (c * v^2)), where
+## phi - angle of rotation during a gravitational maneuver (angle at which the velocity vector of the rocket rotates),
```

Comment:
There are 3 axis (in 3d space) where it can rotate. Is there any way to determine, where it rotates? Also it can rotate in positive and negative direction. How to choose one?

Reply:
> There are 3 axis (in 3d space) where it can rotate. Is there any way to determine, where it rotates? Also it can rotate in positive and negative direction. How to choose one?

I don't quite understand the question. After all, we are talking about a rocket orbiting a planet, not around its axis. Therefore, I do not understand what the three possible axes of rotation and the positive or negative direction have to do with it.

Reply:
Yes, but without a picture it is not clear, how the rocket rotates. You can add to description, that trajectory of a rocket and a planet lies on the same plane. And that rocket rotates towards the planet. Or you can add a link to the picture in a description.